src/ORM/BehaviorRegistry.php

```diff
@@ -84,12 +84,11 @@ public function setTable(Table $table)
     /**
      * Resolve a behavior classname.
      *
-     * Part of the template method for Cake\Core\ObjectRegistry::load()
-     *
      * @param string $class Partial classname to resolve.
      * @return string|false Either the correct classname or false.
+     * @since 3.5.7
      */
-    protected function _resolveClassName($class)
+    public static function className($class)
```

Comment:
Can we have this method return string|null instead?
The former method sure must stay BC, but this new one could get rid of the false code smell IMO.

Reply:
@dereuromark For now maintaining consistency with `App::className()` is more important so that this method can be easily swapped where required. In future both can be changed to return null.

Reply:
Well the basic usage will still be the same: if () and empty() will be the same, but future strictness in nullable regards is easier if the method would already have a proper API.
But no big deal right now either way.

Reply:
This new method could use the string|null pattern. As people switch over they would be able to update their checks as well.